glean-core/ios/Glean/Metrics/Ping.swift

```diff
@@ -25,6 +29,7 @@ public class PingBase {
     var handle: UInt64
     var name: String
 
+    /// Create a new Ping with the given name.
```

Comment:
Again, not sure if we really need it here but this is also missing `- parameters` section.